wiki/Beatmap/Difficulty/fr.md

```diff
@@ -9,6 +9,8 @@ tags:
   - nom de la difficulté
   - classement en étoiles
   - étoiles
+outdated:
+  - true
```

Comment:
these should be `outdated: true` on one line (github on mobile doesn't let me do multiline suggestions..)